features/step_definitions/git-flow.js

```diff
@@ -200,4 +186,22 @@ module.exports = function server() {
             Assert.oneOf(data.state, ['success', 'pending']);
         })
     );
+
+    // eslint-disable-next-line new-cap
+    this.After({
```

Comment:
Do your tests care whether a pipeline was recently created or already exists? Can they tell the difference?